kolibri/plugins/management/assets/src/device_management/views/containers/subpage-container.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+
+  <div class="subpage-container" :class="{ 'side-margin': withSideMargin }">
+    <slot />
+  </div>
+
+</template>
+
```

Comment:
I suspect you are not running with `--lint` otherwise, this would have blown up during build. Also seems like a lot of the linting that is happening is not using prettify.

Reply:
I'm not seeing any issue here